### DISCUSSION TITLE
Raspimatic findet keine neuen Geräte mehr

### DISCUSSION THREAD

**Discussion by Tom1802** [OpenCCU/OpenCCU#2767] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Ich nutze die Raspimatic in der Version 3.53.34.20201121 mit node-red (Homekit). Ich weiß es ist eine ältere Version, aber "never change a running system". Ich habe vor ein paar Monaten eine Unifi-USG Firewall in betrieb genommen und wollte heute das HMIP-WKP anlernen. Trotz mehrfachen Versuchen online oder lokal, ausschalten der Raspimatic und neustarts, wurde kein Geräte gefunden. 

Ich habe den Port 8443 in der Firewall, als auch in der Raspi freigegeben. Gibt es sonst noch bekannte Probleme? 

### Describe the behavior you expected

Anlernen des Geräts

### Steps to reproduce the issue

1.
2.
3.
...
siehe oben

### What is the version this bug report is based on?

3.53.34.20201121

### Which base platform are you running?

rpi3 (RaspberryPi3, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
leider nichts gefunden
```

**Comment by Baxxy13** [comment, marked as answer]
Ein Problem ist zumindest deine alte Version. Das HmIP-WKP wurde mitte 2022 vorgestellt, somit dürfte die Unterstützung auch erst irgendwann 2022 in der CCU3 Firmware gelandet sein.

**Comment by jens-maus** [comment]
In der Tat halte ich es auch mehr als gewagt einen vermeintlichen Fehler in RaspberryMatic selbst via offiziellen Bugreport für eine solch maßlos veraltete RaspberryMatic Version hier einzustellen. Sowas gehört maximal in ein Diskussionsforum denn bugreports/issues melden sollen nicht ohne Grund nur auf aktuellsten Versionen erfolgen! Das war/ist auch so übrigens im Bug/Issuetemplate angemerkt!
